server/services/emailService.ts

```diff
@@ -1,6 +1,9 @@
 import sgMail from "@sendgrid/mail";
 export default class EmailService {
-  constructor() {
+  private apiRoute: string;
+
+  constructor(apiRoute: string = `${process.env.PAGE_URL}api/v1/user`) {
```

Comment:
```suggestion
  constructor(apiRoute: string ) {
```
This should be passed in to the function since there are two assumptions: 
1) The api version
2) the user route - what if other routes want to send emails
Best not to default these types of variables. Further, the `user` part of the route should not be passed in and it should be part of the built string in the function - just `/api/<version>`